src/soilbiogeochem/SoilBiogeochemNitrogenFluxType.F90

```diff
@@ -315,11 +377,129 @@ subroutine InitHistory(this, bounds)
          avgflag='A', long_name='atmospheric N deposition to soil mineral N', &
          ptr_col=this%ndep_to_sminn_col)
 
+    if (use_fan) then
```

Comment:
is there every a sum of all the manure flux inputs that's written to history files?

Reply:
At the moment there isn't.